circom/tests/loops/call_inside_loop.circom

```diff
@@ -27,18 +29,6 @@ template CallInLoop(n, m) {
 
 component main = CallInLoop(2, 3);
 
-//// Use the block labels to check that the loop is NOT unrolled
```

Comment:
Why is the loop not unrolled in this case?

Reply:
Unroller does not check loops inside Circom functions, only templates. I made a JIRA task to improve this: https://veridise.atlassian.net/browse/VAN-672

Reply:
Yeah, right. So you are testing for the for loop in the function then? I did not notice it at first

Reply:
Yes, maybe unnecessary to check for but I erred on the side of too many checks so that the tests more fully showed the current state of things and running the tests would notify us when anything changes.